src/docs/patterns/navigation.mdx

```diff
@@ -22,20 +37,23 @@ Navigation is the way in which users access the different areas of your applicat
 
 ## Fixed Navigation
 
-The most basic navigation panel is fixed in place and always visible on the left side of the application. Note: a fixed navigation panel is not suitable when going to smaller resolutions or mobile devices.
+The most basic navigation panel is fixed in place and always visible on the left side of the application The navigation can be a fixed list of icons with labels or a fixed panel that is open and revealing multiple navigation options Due to internationalization and accessibility standards PX Blue recommends the use of an open panel displaying the navigation options Another benefit of using a fixed navigation is the ability to use expandable tree navigation as well as more area for product branding. Note: A fixed navigation panel is not suitable when the application is viewed on smaller resolutions or mobile devices
 
 ## Collapsible Navigation
 
 A more common implementation of the navigation panel is a collapsible panel that can be opened and closed via the menu icon in the top left corner of the application. This implementation can have two different variations: content-resize or content-cover.
 
+INSERT IMAGE
+
 ### Content-Resize / Content-Push
 
 <ImageGrid images={[PermanentImg]} />
 
-The content-resize implementation does not cover any of the main application content. Rather, the navigation panel slides in and the main application container resizes to accommodate (the navigation panel and main panel are side by side instead of one on top of the other).
+The content-resize implementation does not cover any of the main application content. Rather, the navigation panel slides in and the main application container resizes to accommodate (the navigation panel and main panel are side by side instead of one on top of the other). This pattern should be considered for products with large amounts of data displays and application features.
 
 ### Content-Cover
 
+<ImageGrid images={[ContentCover1, ContentCover2]} regularWidth />
 <ImageGrid images={[TemporaryImg]} />
```

Comment:
Are these sections still relevant? Should we describe them under the variations that they apply to (temporary is content cover, the others are content-push)?

Reply:
Moved to the variations section